cmd/plume/fedoraspecs.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Red Hat Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+type awsPartitionFedoraSpec struct {
```

Comment:
Why do we need to duplicate these structs with a different name?

EDIT: Looks like they are flattened in a later commit, just merge that one down.